l10n_es_facturae_face/views/res_company_view.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="view_company_form">
+        <field name="name">res.company.form</field>
+        <field name="model">res.company</field>
+        <field name="inherit_id" ref="l10n_es_facturae.view_company_form" />
```

Comment:
```suggestion
        <field name="inherit_id" ref="l10n_es_aeat.view_company_aeat_form" />
```
`tax_agency_id` is defined in this view instead

Reply:
Toda la razón, cambio el campo por `facturae_version`